scripts/build_pkg.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -ex
+
+# ensure coverage is turned on
+export CFLAGS="$CFLAGS -fprofile-arcs -ftest-coverage"
+export LDFLAGS="$LDFLAGS -fprofile-arcs"
+
+if [[ $ABI = 32 ]]; then
+    export CFLAGS="$CFLAGS -m32"
+    export LDFLAGS="$LDFLAGS -m32"
+fi
+
+# build this package
+if [[ -x autogen.sh ]]; then
+    ./autogen.sh
+    ./configure --with-gaproot=$GAPROOT
+    make -j4 V=1
+elif [[ -x configure ]]; then
+    ./configure $GAPROOT
+    make -j4
+fi
+
+# trick to allow the package directory to be used as a GAP root dir
+ln -s . pkg
```

Comment:
If this our latest standard setup (I mean, includes recent overhaul by @fingolfin) you need to look at this change that I would like to apply to it: https://github.com/gap-packages/example/pull/15/commits/786673dfaa153c5cf7dc2ac1d6e1bfbaec6e03a6